openapiSpec.yaml

```diff
@@ -0,0 +1,832 @@
+openapi: 3.0.2
+info:
+    title: RTR app
+    description:  The intended API calls for our QCode applications are shown in this page
+    version: 1.0.0
+paths:
+    /solutions:
+        post:
+            operationId: createSolution
+            tags:
+                - Solution
+            summary: creates a new solution
+            requestBody:
+                required: true
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#/components/schemas/Solution'
+            responses:
+                '201':
+                    description: Solution created succesfully
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/Solution'
+                '400':
+                    description: Not a valid request
+                default:
+                    description: Unexpected error
+        get: 
+            operationId: getSolutions
+            tags:
+                - Solution
+            summary: get a list of solutions (with nested users)
+            parameters:
+                -   name: q
+                    in: query
+                    description: if present search for this query
+                    schema: 
+                        type: string
+                -   name: pageNumber
```

Comment:
was dit niet sectionNumber?

Reply:
het plan was om het terug te veranderen naar pageNumber en pageSize omdat we nu met pagina's werken ipv scroll secties

Reply:
Aight dan moet ik dat ook even veranderen in mijn call van search